app/controllers/posts/comments_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Posts::CommentsController < ApplicationController
+  before_action :set_post
+
+  def new
+    @comment = @post.comments.build(user: current_user)
+  end
+
+  def create
+    @comment = @post.comments.build(comment_params)
+    @comment.user = current_user
+    if @comment.save
+      redirect_to post_path(@post), notice: t('controller.created')
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  private
+
+  def set_post
+    @post = Post.find(params[:post_id])
+  end
+
+  def comment_params
+    params.require(:comment).permit(:body)
```

Comment:
```suggestion
    params.require(:comment).permit(:body).merge(user_id: current_user.id)
```

とすると、createのところが

```rb
@comment = @post.comments.build(comment_params)
```
これだけでいけるかも

Reply:
@uekkie 
ハッシュ要素だからmergeでいけるんですね！
current_userのidも含めると単なるparamsの値だけではなくなるので、他のメソッド名が良さそうですかね？？

Reply:
createアクションのところでmergeしてもよさそう